db/migrations/00014_create_stored_functions.sql

```diff
@@ -8,7 +8,7 @@ SELECT exists(SELECT 1
                        INNER JOIN eth.state_cids ON (storage_cids.state_id = state_cids.id)
                        INNER JOIN eth.header_cids ON (state_cids.header_id = header_cids.id)
               WHERE storage_path = path
-                AND block_number > height
+                AND block_number >= height
```

Comment:
If this needs changed the comment for the function should be updated too

Reply:
Done